server/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM ruby:2.6.5
 
 
 RUN apt-get update \
-    && apt-get install -y --no-install-recommends nodejs libpq-dev
+    && apt-get install -y --no-install-recommends nodejs libpq-dev libxslt1-dev libffi-dev postgresql-client-common postgresql-client
```

Comment:
I think that we need postgresql-client 

The others are listed as included in other packages. postgresql-client-common is part of postgresql-clent and libxslt1-dev is part of nokogiri. 

Shall we start by adding postgresql-client?